plugin/adsk/plugin/plugin.cpp

```diff
@@ -134,6 +135,8 @@ MStatus initializePlugin(MObject obj)
         }
     }
 
+    UsdMayaSceneResetNotice::InstallListener();
```

Comment:
This was already being installed by the Pixar plugin, but MayaUSD should also be using it.